cfgov/paying_for_college/disclosures/urls.py

```diff
@@ -25,8 +27,12 @@
                 template_name="paying_for_college/disclosure_technote.html"),
             name="pfc-technote"),
 
-    re_path(r"^api/search-schools.json", school_search_api,
-            name="school_search"),
+    flagged_re_path(
+        'ELASTICSEARCH_DSL_PFC',
+        r"^api/search-schools.json",
```

Comment:
Unrelated but for my understanding - why do we have a file type ending in a route? Does this return a file?

Reply:
@higs4281 Do you recall if there is a file type ending in a route that returns a file?